plugins/che-dummy-plugin/0.0.1/meta.yaml

```diff
@@ -1,8 +1,8 @@
 id: che-dummy-plugin
 version: 0.0.1
-type: Che Plugin   # can be `Che Plugin` or `Che Editor`
-name: che-java
-title: Che Java Plugin
-description: ...
-icon: https://....
-url: https://.... # this points to the URL of the tar.gz of the plugin (usually this is the plugin release URL on github)
+type: Che Plugin
+name: Che dummy plugin
+title: Che dummy Plugin
+description: A hello world theia plug-in wrapped into a Che Plug-in
+icon: https://www.eclipse.org/che/images/ico/16x16.png
+url: https://github.com/ws-skeleton/che-dummy-plugin/releases/download/untagged-8d788fa8defd39f5ffd7/che-dummy-plugin.tar.gz
```

Comment:
do we have a way to specify "latest" ?
for example https://github.com/ws-skeleton/che-dummy-plugin/releases/latest
is redirecting me to https://github.com/ws-skeleton/che-dummy-plugin/releases/tag/untagged-8d788fa8defd39f5ffd7

Reply:
or how to automate updating entries there each time a plug-in is released

Reply:
@garagatyi thoughts?

Reply:
no, I thought I had accepted it